edb/edgeql/compiler/stmtctx.py

```diff
@@ -116,6 +117,27 @@ def fini_expression(
     ):
         ir = setgen.scoped_set(ir, ctx=ctx)
 
+    # Make sure that all materialized sets have their views compiled
+    v = ast_visitor.NodeVisitor()
+    v.node_visit(ir)
+    children = [x for x in v.memo if isinstance(x, irast.Stmt)]
```

Comment:
Why not `ast.find_children`?

Reply:
I was going to say it was because find_children doesn't properly handle multiple levels of nested data, and before I created MaterializedSet, I just stored that tuple in a dict.
But *neither* looks at dict *values*, so that isn't it either.

I think maybe I needed it for some debugging I was trying to do?
In any case, it can be `find_children`.